src/components/sidebar/cart/Cart.js

```diff
@@ -24,14 +26,27 @@ function Cart({ classes, sections }) {
   );
 
   const label = uniqueSections.length === 1 ? 'Class' : 'Classes';
+  function displayButton(length) {
+    if (length !== 0) {
+      return (
+        <Button>
+           Remove All
+        </Button>
+      );
+    }
+    return (null);
+  }
 
   return (
     <React.Fragment>
-      <Typography variant="h5" className={classes.cartHeading}>
-        {uniqueSections.length}
-        {' '}
-        {label}
-      </Typography>
+      <Grid container direction="row" justify="space-between">
+        <Typography variant="h5" className={classes.cartHeading}>
+          {uniqueSections.length}
+          {' '}
+          {label}
+        </Typography>
+        {displayButton(uniqueSections.length)}
```

Comment:
I'm thinking it might be easier and cleaner to do 
`{ uniqueSections.length && <Button> Remove All </Button> }`
or whatever that syntax is since the component itself isn't really complex

Reply:
If using { uniqueSections.length && <Button> Remove All </Button> }, there will be a 0 showing when there is no class in the cart. So I instead used an if else to do it. See my new commit

Reply:
You're right about that, that's my bad. You can do `!!uniqueSections.length` because 0 is a falsy value, see here: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
Do I have to change it to !!uniqueSections.length, or can I keep it as the current if else?

Reply:
I would - no need for the ternary operator, you don't have to return `null`, you can just return nothing using &&

Reply:
makes sense! Will do it now